buildSrc/src/main/java/baritone/gradle/task/BaritoneGradleTask.java

```diff
@@ -43,28 +44,43 @@
             ARTIFACT_STANDARD         = "%s-%s.jar",
             ARTIFACT_UNOPTIMIZED      = "%s-unoptimized-%s.jar",
             ARTIFACT_API              = "%s-api-%s.jar",
-            ARTIFACT_STANDALONE       = "%s-standalone-%s.jar";
+            ARTIFACT_STANDALONE       = "%s-standalone-%s.jar",
+            ARTIFACT_FORGE_UNOPTIMIZED = "%s-unoptimized-forge-%s.jar",
```

Comment:
staggering of the = signs is a bad meme, either do it right or dont do it at all tbh...